signer/src/stacks/contracts.rs

```diff
@@ -1445,7 +1445,7 @@ impl RotateKeysErrorMsg {
 
 /// A wrapper type for smart contract deployment that implements
 /// AsTxPayload.
-#[derive(Clone, Copy, Debug, Hash, PartialEq)]
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, strum::Display)]
```

Comment:
nit: I would also add
```rust
#[strum(serialize_all = "snake_case")]
```
or
```rust
#[strum(serialize_all = "SCREAMING_SNAKE_CASE")]
```
but this is all for logging at the moment so whatever is fine. But if we were to serialize this to a string in to the database then we should use `snake_case`.